internal/show/brokers.go

```diff
@@ -51,14 +51,19 @@ func Brokers(clusterInfo *cluster.Info, status reporter.Interface) bool {
 		return true
 	}
 
-	fmt.Printf(template, "NAMESPACE", "NAME", "COMPONENTS")
+	fmt.Printf(template, "NAMESPACE", "NAME", "COMPONENTS", "DEFAULT CUSTOM DOMAINS", "DEFAULT GLOBALNET CLUSTER SIZE",
+		"GLOBALNET CIDR RANGE", "GLOBALNET ENABLED")
 
 	for i := range brokers {
 		fmt.Printf(
 			template,
 			brokers[i].Namespace,
 			brokers[i].Name,
 			strings.Join(brokers[i].Spec.Components, ", "),
+			brokers[i].Spec.DefaultCustomDomains,
```

Comment:
This is an array as well.

```suggestion
			strings.Join(brokers[i].Spec.DefaultCustomDomains, ", "),
```